Integrations/python/deephaven/Types/__init__.py

```diff
@@ -0,0 +1,74 @@
+# 
```

Comment:
I have some angst about calling this `Types` because the name is not pythonic.  We have used `TableTools` and such because those are autogenerated classes that map to Java objects.  You could argue that using caps was a bad idea.
@jmao-denver what is our religion?  Are we going with caps and just being different?